keanu-project/src/test/java/io/improbable/keanu/vertices/dbl/DoubleVertexSamplesTest.java

```diff
@@ -14,10 +14,10 @@
 
 public class DoubleVertexSamplesTest {
     List<DoubleTensor> values = ImmutableList.of(
-        DoubleTensor.create(new double[]{0, 16, 4}, new long[]{1, 3}),
-        DoubleTensor.create(new double[]{-4, -8, 4}, new long[]{1, 3}),
-        DoubleTensor.create(new double[]{8, -4, 12}, new long[]{1, 3}),
-        DoubleTensor.create(new double[]{4, 4, 8}, new long[]{1, 3})
+        DoubleTensor.create(new double[]{0, 16, 4}, 1, 3),
+        DoubleTensor.create(new double[]{-4, -8, 4}, 1, 3),
+        DoubleTensor.create(new double[]{8, -4, 12}, 1, 3),
+        DoubleTensor.create(new double[]{4, 4, 8}, 1, 3)
```

Comment:
Just curious, why have you changed this? I think I find explicitly creating an array for shape more readable (but it's super subjective, of course).

Reply:
Yeah, I don't have a strong opinion on it - I kind of did it because intellij flagged is redundant

Reply:
Oh, ok, I thought that there might be some more IntelliJ wisdom going on :)